themes/radial/components/code-blocks.typ

```diff
@@ -0,0 +1,41 @@
+#import "../colors.typ": *
+
+#let raw_not_block = box.with(
+  fill: surface_2,
+  inset: (x: 3pt, y: 0pt),
+  outset: (y: 3pt),
+  radius: 2pt,
+)
+
+#let raw_block(it) = {
+  set par(justify: false);
+  // the line counter
+  let i = 0;
+  let box_radius = 1.5pt;
+
+  let detail_radius = 1.5pt;
```

Comment:
Two variables with the same name but the only difference is the format. This should be changed.